docs/application/dotnet/guides/internationalization/localization.md

```diff
@@ -0,0 +1,252 @@
+# Localization
+
+This document describes how to localize your Tizen Xamarin.Forms application.
+
+- [String Localization](#string-localization)
+- [Display the correct Language](#display-the-correct-language)
+- [Image Localization](#image-localization)
+- [Application Name Localization](#application-name-localization)
+
+
+## String Localization
+You can begin with reading the following how-to article of Xamarin.Forms Localization.
+
+For more information on String Localization, see [Xamarin.Forms Localization](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/localization/#string-and-image-localization)
+
+1. Add the `RESX` resource files that will be used to store all the text used in the your application.
+![local_resx](media/local_resx.png)
+![local_appresources_resx1](media/local_appresources_resx1.png)
+
+2. Change the string visibility from internal to public. Select your `resx` file and click **Properties**. In the Configuration Properties, change Custom Tool to PublicResXFileCodeGenerator as shown in the following image:
+![local_appresources_resx2](media/local_appresources_resx2.png)
+![local_appresources_resx3](media/local_appresources_resx3.png)
+
+3. Add language-specific resource files, which must follow a specific naming convention and use the same filename as the base resources file.
+![local_appresources_resx4](media/local_appresources_resx4.png)
+
+- Use name of texts with String type in the `RESX` files in your user interface code.
```

Comment:
The layout is not clear here.

After numbered list, this "in user interface code" file and "in the xaml code" items are just listed.

Are these sub item of 3rd list item above? Would you please cleare up?

Reply:
No, numbered list is to make res file. this "in user interface code" file and "in the xaml code" items are code to use the res file.